java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -914,6 +914,13 @@ private String getBlockName(Block b) {
         if (b != null) {
             String sName = b.getSystemName();
             String uName = b.getUserName();
+            // Jay Janzen
```

Comment:
@rhwood,

I don't believe that this is an issue that should ever have warranted threatening to reject the PR.

The fact is this doesn't change the function of the underlying code contained within the PR, and only a little bit of the UI.

Paul